operator/cmd/mesh/manifest-generate_test.go

```diff
@@ -93,6 +93,18 @@ func TestManifestGenerateFlags(t *testing.T) {
 			flags:      "-s profile=minimal",
 			noInput:    true,
 		},
+		{
+			desc:       "flag_set_includeIPRanges_single",
+			diffIgnore: "ConfigMap:*:istio",
+			flags:      "--set values.global.proxy.includeIPRanges=172.30.0.0/16",
```

Comment:
can we somehow avoid adding 20,000 lines for this PR?

Reply:
@howardjohn This PR is XXL because I wrote two test cases and used `UPDATE_GOLDENS=true go test -run TestManifestGenerateFlags` to generate expected output.  There were no test cases for commas.

- Are you suggesting I remove the non-comma case?  No problem.
- Are you suggesting I find a way to make the comma case smaller???

Reply:
Can we just integrate it into another test case possibly? I don't expect you to solve the "these tests are way too verbose problem" right here, but someone should. Its a pretty bad dev ux when I make a 1 line change and suddenly 100k lines are updated

Reply:
OK; done.